info/v1/container.go

```diff
@@ -306,6 +306,11 @@ type MemoryStats struct {
 	// Units: Bytes.
 	Usage uint64 `json:"usage"`
 
+	// Various statistics reported in memory.stat in cgroupfs. See the section
```

Comment:
We already include some of the stats from `memory.stat`. For example `WorkingSet` is derived using data from `memory.stat`.
Can we instead expose specific fields? Explicit fields will help with understanding of the API as well.